### ISSUE TITLE
how to return a normal Response if with format=xlsx

### ISSUE THREAD

**Issue by wgf4242** [wq/django-rest-pandas#38]
If there isn't a 'form' param in request ,I want return an error, but raised an error. 
```python
def list(self, request: Request, *args: Any, **kwargs: Any) -> Response:
    if self.request.query_params.get('form') and self.request.query_params.get('format') == 'xlsx':
        return Response({'error' : 'no form'})
```

error:

`Response data is a dict, not a DataFrame!`

**Comment by sheppard** [comment]
Yes, by default the `Response` object is rendered with the negotiated renderer, regardless of the content.  Here are some things you could try:

1. Throw an exception instead of returning a `Response`
2. Explicitly override the "Content-Type" header to be "application/json" (perhaps using Django's `HttpResponse`)
3. Generate a small DataFrame with a single row containing the error message (`pd.DataFrame([{"error": no form"}]`)

**Comment by thmxluis** [comment]
![image](https://user-images.githubusercontent.com/8526986/87480873-d5a05200-c5f3-11ea-9665-dfc62c06942b.png)

Hello, could you help me? I am trying to generate an excel and everything is fine but it is not downloaded, it is only generated in the browser or does anyone have any idea why?

![image](https://user-images.githubusercontent.com/8526986/87481117-51020380-c5f4-11ea-8c5b-099b1b71067f.png)
